### PR TITLE
Editorial

### PR DESCRIPTION
**Pull request by b---c**
As mentioned in #138, this is an attempt to consolidate and isolate the editorial improvements from a number of PRs

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -861,10 +856,9 @@ Figure: HTTP 401 Response to a Protected Resource Request with an Invalid Token
 
 Protected resources simultaneously supporting both the `DPoP` and `Bearer` 
 schemes need to update how evaluation of bearer tokens is performed to prevent 
-downgraded usage of a DPoP-bound access tokens. 
-Specifically, such a protected resource MUST reject an access
-token received as a bearer token per [@!RFC6750], if that token is
-determined to be DPoP-bound. 
+downgraded usage of a DPoP-bound access tokens.
```
Comment (selfissued) [inline main.md:859]: access tokens -> access token.

File: main.md [hunk 2]
```diff
@@ -885,16 +879,17 @@ prolonged deployments of protected resources with mixed token type support.
 
 # Authorization Server-Provided Nonce {#ASNonce}
 
+This section specifies a mechanism using opaque nonces provided by the server
+that can be used to limit the lifetime of DPoP proofs.
+Without employing such a mechanism, a malicious party controlling the client
+(including potentially the end user)
```
Comment (selfissued) [inline main.md:885]: end user -> end-user

**Review by selfissued — APPROVED** [review]
Please address my two minor comments, then good go go!  Thanks for doing this!

File: main.md (hunk 1 above)
Comment (b---c) [reply main.md]: and done f51a9d998120a95c0f6e8fa64195f566293ab7e2

File: main.md [hunk 3]
```diff
@@ -397,21 +398,19 @@ HTTP request (see also (#request_integrity)).
...
-    header of the JWT,
- 1. the `jwk` header of the JWT does not contain a private key,
- 1. the `htm` claim matches the HTTP method value of the HTTP
-    request in which the JWT was received,
+    JOSE header parameter,
+ 1. the `jwk` JOSE header parameter does not contain a private key,
+ 1. the `htm` claim matches the HTTP method of the current request,
  1. the `htu` claim matches the HTTPS URI value for the HTTP
     request in which the JWT was received, ignoring any query and
     fragment parts,
```
Comment (ioggstream) [inline main.md:416]:
```suggestion
 1. the `htu` claim matches the https URI of the HTTP
    request in which the JWT was received, ignoring any query and
    fragment parts,
```

iirc the protocol is `HTTP`. The URI scheme is `https`. 

Probably a more precise statement would be

> matches the target URI [@RFC7231, section 2] of the current request
> ignoring any query and fragment parts, and with the https URI scheme

Consider that the latest HTTP semantics disentangles the request URI from 
HTTP messaging (see https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#rfc.section.7.1.p.2)

File: main.md [hunk 4]
```diff
@@ -339,20 +340,20 @@ The payload of a DPoP proof contains at least the following claims:
...
+   fragment parts.
+ * `iat`: Creation timestamp of the JWT ([@RFC7519, section 4.1.6]).
 
 When the DPoP proof is used in conjunction with the presentation of an access token, see 
 (#protected-resource-access), the DPoP proof MUST also contain the following claim:
 
-* `ath`: hash of the access token (REQUIRED).
+* `ath`: hash of the access token.
    The value MUST be the result of a base64url encoding (with no padding) the SHA-256
    hash of the ASCII encoding of the associated access token's value.
```
[the remark this answers is not in the document]
Comment (bc-pi) [reply main.md:354]: yes, just the access token

[the remark this answers is not in the document]
Comment (bc-pi) [reply main.md:354]: base64url can/should come from RFC7515 https://datatracker.ietf.org/doc/html/rfc7515#section-2 which is admittedly a kinda strange place for it but more precise for this usage

**Pull request merged**